apps/web/src/services/asset-inventory/store/collector-form-store.ts

```diff
@@ -10,10 +13,15 @@ type CollectorFormAction = any;
 export const useCollectorFormStore = defineStore<string, CollectorFormState, CollectorFormGetter, CollectorFormAction>('collector-form', {
     state: () => ({
         originCollector: null,
+        attachedServiceAccount: null,
     }),
     actions: {
         setOriginCollector(collector: any) {
             this.originCollector = collector;
         },
+        setAttachedServiceAccount(serviceAccount: AttachedServiceAccount) {
+            if (!serviceAccount || !serviceAccount.length) this.attachedServiceAccount = null;
```

Comment:
nitpick: I think we can shorten this. :)

```suggestion
            if (!!serviceAccount?.length) this.attachedServiceAccount = null;
```